eppo_core/src/eval/eval_rules.rs

```diff
@@ -14,128 +15,97 @@ impl Rule {
         visitor: &mut V,
         attributes: &Attributes,
     ) -> bool {
-        self.conditions
-            .iter()
-            .all(|condition| condition.eval(visitor, attributes))
+        self.conditions.iter().all(|condition| match condition {
+            TryParse::Parsed(condition) => condition.eval(visitor, attributes),
+            TryParse::ParseFailed(raw_condition) => {
+                visitor.on_condition_skip(raw_condition);
+                false
```

Comment:
ok cool so by returning false we won't match when a condition is skipped due to failing to parse 👌